clients/client-python/gravitino/filesystem/gvfs_storage_handler.py

```diff
@@ -371,7 +371,7 @@ def _get_most_suitable_credential(self, credentials: List[Credential]):
         return None
 
     def _get_last_modified(self, entry: Dict):
-        return entry["LastModified"]
+        return entry["LastModified"] if "LastModified" in entry else None
```

Comment:
When will `LastModified` be absent?

Reply:
In our lastest test, the key can't be found if it's a directory not an object.